src/commands/php-errors.php

```diff
@@ -73,16 +73,43 @@ protected function execute( InputInterface $input, OutputInterface $output ) {
         $output->writeln( "<comment>Adding temporary bot collaborator to $site_domain.</comment>" );
 
         $add_collaborator_request = $api_helper->call_pressable_api(
-                "sites/{$pressable_site->id}/collaborators",
+                "collaborators/batch_create",
                 'POST',
                 array(
-                    'siteId' => $pressable_site->id,
-                    'email' => PRESSABLE_BOT_COLLABORATOR_EMAIL,
+                    'siteIds' => array(
+                        $pressable_site->id,
+                    ),
+                    'email'   => PRESSABLE_BOT_COLLABORATOR_EMAIL,
+                    'roles'   => 'sftp_access',
                 )
         );
 
-        if ( empty( $add_collaborator_request->data->accountId ) ) {
-            $output->writeln( "<error>Failed to create a temporary bot collaborator. Aborting!</error>" );
+        if ( ! is_null( $add_collaborator_request->errors ) ) {
+            $output->writeln( "<error>Error creating temporary bot collaborator. Aborting!</error>" );
+            exit;
+        }
+        // Get new collaborator id.
+        $max_tries = 0;
```

Comment:
This variable should probably be renamed, it makes it sound like the max number of tries allowed is zero. I'd rename it to just `tries` or `num_tries`.

```suggestion
        $tries = 0;
```